src/content/site/rules.md

```diff
@@ -7,74 +7,74 @@ path: /rules
 - [General Rules](#general-rules)
 - [Golden Rules](#golden-rules)
 - [Content Policy](#content-policy)
-- [Why isn't security talk allowed?](#Why-dont-you-allow-talk-regarding-hacking)
+- [Why isn't security talk allowed?](#why-dont-you-allow-talk-regarding-hacking)
 - [Server Roles](#roles)
-- [Nickname Policy](#Nickname-Policy)
-- [What is Modmail / How to contact staff](#What-is-ModMail)
+- [Nickname Policy](#nickname-policy)
+- [What is Modmail / How to contact staff](#what-is-modmail)
 
 # General Rules
 
 Below you will find the rules of the server. These are the do's and don'ts of being here. Follow them if you wish to stay, ignorance is not an excuse.
 
-**1. Do not advertise.**
+**1. Do not advertise.**  
```

Comment:
It's not very obvious that the trailing whitespaces are actually causing a break. Can we switch to `<br />`? (it works)

Reply:
Makes sense - I'll look at this later when I get home and make the changes :)

Reply:
All good, otherwise I will if I have time

Reply:
Fixed now :) I also added `.` to the titles that were missing them for consistency.